doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -261,7 +261,18 @@ Debug samples
 DECT NR+ samples
 ----------------
 
-|no_changes_yet_note|
+* :ref:`dect_shell_application` sample:
+
+  * Added:
+
+    * The ``dect mac`` command.
+      A brief MAC-level sample on top of DECT PHY interface with new commands to create a periodic cluster beacon, scan for it, associate or disassociate a PT/client, and send data to a FT/beacon random access window.
+      This is not a full MAC implementation and not fully compliant with DECT NR+ MAC specification (`ETSI TS 103 636-4`_).
+
+  * Updated:
+
+    * The ``dect rssi_scan`` command in addition to the default high/low ``rssi_scan`` command for busy/possible/free subslot count-based RSSI scan.
```

Comment:
This sentence doesn't read nicely. The part that has been added is "busy/possible/free subslot count-based RSSI scan" to my knowledge.

See if this is better in your opinion:
```suggestion
    * The ``dect rssi_scan`` command with busy/possible/free subslot count-based RSSI scan in addition to the existing high/low RSSI scan.
```
Or maybe even leave out "in addition to..." part.

If not modifying otherwise "for" -> "with" in original sentence makes it clearer for me and I'd even question is "for" saying what we want to say but that's @peknis area to judge. Anyway, having the existing functionality mentioned first takes the focus on what's really the key thing added.

Reply:
Agree with Tommi. Leave out the "in addition to the...". With those adjustment, the focus should be right on spot.

Reply:
left out as no thumb up from Pekka :-)